docs/registries-authentication.md

```diff
@@ -20,10 +27,10 @@ cat <<EOF | tee aa-offline_fs_kbc-resources.json
 }
 EOF
 ```
-- Add `AA_KBC="offline_fs_kbc"` prior to the `make image` step
```

Comment:
The reason i added it this way is because instructions are located on different file, so i'd suggest to give context by switching "run command" with "build image with" or "make sure to build image with"